targets/TARGET_Realtek/TARGET_AMEBA/us_ticker.c

```diff
@@ -23,24 +23,33 @@
 #define SYS_TIM_ID      1   // the G-Timer ID for System
 #define APP_TIM_ID      6   // the G-Timer ID for Application
 
+#define TICK_TO_US(x)     (uint64_t)(((x)/2) * 61 + ((x)%2) * TIMER_TICK_US)
```

Comment:
An explanation of where '61' comes from would be useful here

Reply:
@adbridge Clock source is 32K, therefore, us per tick is 30.5. The TICK_TO_US macro is basically:  (tick / 2) * (30.5*2) + residue 

This patch is basically broken. I have submitted another PR #5143 to address us ticker issue, and also added proper inline comment.